site/content/docs/5.2/forms/overview.md

```diff
@@ -160,6 +160,67 @@ Use the `.is-required` class on `label` to add a `*` symbol after it.
 </form>
 {{< /example >}}
 
+## Form helper
+
+Provide some help on an input with `.btn-helper`. **Be sure to include text for screen readers**, as we've done with `<span class="visually-hidden">`.
+
+On top on this button, apply a [popover]({{< docsref "/components/popovers" >}}) to display the help. A [tooltip]({{< docsref "/components/tooltips" >}}) can also be applied for small and simple text helpers without any link, button, etc...
+
+{{< example >}}
```

Comment:
I don't have any ideas right now but it is very strange to display those 2 exact same buttons like this. On top of that, they have a pre-defined margin-left which make them look weird.
I'm wondering if we couldn't just keep the real-life examples.